playwright/tests/eventListing.spec.ts

```diff
@@ -9,20 +9,8 @@ test.describe('eventListing', () => {
     await expect(page).toHaveURL(/\/events\//)
   })
 
-  test('Event Listing widget changes form fields based on selection', async ({
```

Comment:
This is testing functionality internal to the widget, which is sourced from vets-website. It can be tested there; we do not need to be testing its functionality here.